examples/platform/alerts.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2023. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""
+Example script illustrating retrieval of container runtime alerts.
```

Comment:
What is this for as it mentions container_runtime but calls CB_ANALYTICS

Reply:
This was me creating a separate set of test examples to run off of. May be expanded or removed in a future release. probably shouldn't be in the repo yet. Will keep on local for now